packages/oidc-middleware/src/oidcUtil.js

```diff
@@ -74,13 +74,23 @@ oidcUtil.bootstrapPassportStrategy = context => {
     },
     sessionKey: context.options.sessionKey,
     client: context.client
-  }, (tokenSet, userinfo, done) => {
-    return tokenSet && userinfo
-      ? done(null, {
-        userinfo: userinfo,
-        tokens: tokenSet
-      })
-      : done(null);
+  }, (...args) => {
```

Comment:
it looks like doing rest parameters here is going to make [userinfo check](https://github.com/panva/node-openid-client/blob/master/lib/passport_strategy.js#L155) evaluate into `false` (i.e. `this._verify.length == 0`), so the users who have userinfo enabled won't be able to fetch it